src/components/ReportActionItem/ReportPreview.js

```diff
@@ -84,11 +88,27 @@ const defaultProps = {
 };
 
 function ReportPreview(props) {
-    const managerID = props.iouReport.managerID || 0;
+    const managerID = props.iouReport.managerID || props.action.actorAccountID || 0;
     const isCurrentUserManager = managerID === lodashGet(props.session, 'accountID');
-    const reportAmount = CurrencyUtils.convertToDisplayString(ReportUtils.getMoneyRequestTotal(props.iouReport), props.iouReport.currency);
+    let reportAmount = ReportUtils.getMoneyRequestTotal(props.iouReport);
+    if (reportAmount) {
+        reportAmount = CurrencyUtils.convertToDisplayString(reportAmount, props.iouReport.currency);
+    } else {
+        // If iouReport is not available, get amount from the action message (Ex: "Domain20821's Workspace owes $33.00" or "paid ₫60" or "paid -₫60 elsewhere")
+        reportAmount = '';
+        const actionMessage = _.size(props.action.message) ? props.action.message[0].text || '' : '';
+        const splits = actionMessage.split(' ');
+        for (let i = 0; i < splits.length; i++) {
+            if (/\d/.test(splits[i])) {
+                reportAmount = splits[i];
+            }
+        }
```

Comment:
What if the user adds a number in the comment? That'd replace the amount, wouldn't it? I think we could achieve the same result without the for loop, by using the first match in this regex instead:

```
\S+\d[\d,\.]*?\b
```

Reply:
I don't think the message in the preview action contains any user comment - And if you see the 3 examples the previous comment, this wouldn't work:
```
'Domain20821\'s Workspace owes $33.00'.match(/\S+\d[\d,\.]*?\b/)
['Domain20821', index: 0, input: "Domain20821's Workspace owes $33.00", groups: undefined]
```

Reply:
This feels like a temporary solution for sure, its probably quite brittle with different workspace names and also different currencies, I think we need to ensure any report preview which is shown to the user has the associated iou report loaded in the app. But that can be a polish